mender-update/daemon/state_test.cpp

```diff
@@ -2189,66 +2355,24 @@ vector<StateTransitionsTestCase> GenerateStateTransitionsTestCases() {
-					"ArtifactRollback",
-					"ArtifactRollback_Leave_00",
-					"ArtifactRollbackReboot_Enter_00",
-					"ArtifactRollbackReboot",
-					"ArtifactVerifyRollbackReboot",
-					"ArtifactRollbackReboot_Leave_00",
-					"ArtifactRollbackReboot_Enter_00",
-					"ArtifactRollbackReboot",
-					"ArtifactVerifyRollbackReboot",
-					"ArtifactRollbackReboot_Leave_00",
-					"ArtifactRollbackReboot_Enter_00",
-					"ArtifactRollbackReboot",
-					"ArtifactVerifyRollbackReboot",
-					"ArtifactRollbackReboot_Leave_00",
-					"ArtifactRollbackReboot_Enter_00",
-					"ArtifactRollbackReboot",
-					"ArtifactVerifyRollbackReboot",
-					"ArtifactRollbackReboot_Leave_00",
-					"ArtifactRollbackReboot_Enter_00",
-					"ArtifactRollbackReboot",
-					"ArtifactVerifyRollbackReboot",
-					"ArtifactRollbackReboot_Leave_00",
-					"ArtifactRollbackReboot_Enter_00",
-					"ArtifactRollbackReboot",
-					"ArtifactVerifyRollbackReboot",
-					"ArtifactRollbackReboot_Leave_00",
-					"ArtifactRollbackReboot_Enter_00",
-					"ArtifactRollbackReboot",
-					"ArtifactVerifyRollbackReboot",
-					"ArtifactRollbackReboot_Leave_00",
-					"ArtifactRollbackReboot_Enter_00",
-					"ArtifactRollbackReboot",
-					"ArtifactVerifyRollbackReboot",
-					"ArtifactRollbackReboot_Leave_00",
-					"ArtifactRollbackReboot_Enter_00",
-					"ArtifactRollbackReboot",
-					"ArtifactVerifyRollbackReboot",
-					"ArtifactRollbackReboot_Leave_00",
-					"ArtifactRollbackReboot_Enter_00",
-					"ArtifactRollbackReboot",
-					"ArtifactVerifyRollbackReboot",
-					"ArtifactRollbackReboot_Leave_00",
-					"ArtifactRollbackReboot_Enter_00",
-					"ArtifactRollbackReboot",
-					"ArtifactVerifyRollbackReboot",
-					"ArtifactRollbackReboot_Leave_00",
-					"ArtifactRollbackReboot_Enter_00",
-					"ArtifactRollbackReboot",
-					"ArtifactRollbackReboot_Leave_00",
+					"Download_Enter_00",         "Download",
+					"Download_Leave_00",         "ArtifactInstall_Enter_00",
+					"ArtifactInstall",           "ArtifactInstall_Leave_00",
+					"ArtifactReboot_Enter_00",   "ArtifactReboot",
+					"ArtifactVerifyReboot",      "ArtifactReboot_Error_00",
+					"ArtifactRollback_Enter_00", "ArtifactRollback",
+					"ArtifactRollback_Leave_00", "ArtifactRollbackReboot_Enter_00",
+					"ArtifactRollbackReboot",    "ArtifactVerifyRollbackReboot",
+					"ArtifactRollbackReboot",    "ArtifactVerifyRollbackReboot",
+					"ArtifactRollbackReboot",    "ArtifactVerifyRollbackReboot",
+					"ArtifactRollbackReboot",    "ArtifactVerifyRollbackReboot",
+					"ArtifactRollbackReboot",    "ArtifactVerifyRollbackReboot",
+					"ArtifactRollbackReboot",    "ArtifactVerifyRollbackReboot",
+					"ArtifactRollbackReboot",    "ArtifactVerifyRollbackReboot",
+					"ArtifactRollbackReboot",    "ArtifactVerifyRollbackReboot",
+					"ArtifactRollbackReboot",    "ArtifactVerifyRollbackReboot",
+					"ArtifactRollbackReboot",    "ArtifactVerifyRollbackReboot",
+					// TODO - Missing ArtifactRollback_Leave_00
```

Comment:
Why? It is listed above.

Reply:
I was thinking of after the last `ArtifactVerifyRollbackReboot`